### PR TITLE
Add Target and TargetManager

### PR DESCRIPTION
**Pull request by penguin-s**

### CODE REVIEW

File: src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java [hunk 1]
```diff
@@ -286,6 +291,31 @@ private void runCommandSortExpense(ConsoleCommandSortExpense commandSortExpense)
...
+        targetStr += "Name          : " + target.getName() + "\n";
+        targetStr += "Date and Time : " + target.getDateTime().format(dateTimeFormat) + "\n";
+        targetStr += "Description   : " + target.getDescription() + "\n";
+        targetStr += "Amount        : " + target.getAmount() + "\n";
+        targetStr += "Current Amount: " + target.getCurrentAmount() + "\n";
+        printInformationalMessage(targetStr);
+
+        printInformationalMessage(Messages.CONSOLE_MESSAGE_COMMAND_ADD_TARGET_SUCCESS);
+
+        //saveToFile(targetManager.getTargets());
```
Comment (xzynos) [inline src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java:316]: Do consider using a Todo comment instead if a function is yet to be implemented.

File: src/main/java/seedu/moneygowhere/parser/ConsoleParser.java [hunk 2]
```diff
@@ -391,14 +484,16 @@ private static ConsoleCommandSortExpense parseCommandSortExpense(String argument
      * @throws ConsoleParserCommandDeleteExpenseInvalidException   If the command delete-expense is invalid.
      * @throws ConsoleParserCommandEditExpenseInvalidException     If the command edit-expense is invalid.
      * @throws ConsoleParserCommandSortExpenseInvalidTypeException If the command sort-expense is invalid.
+     * @throws ConsoleParserCommandAddTargetInvalidException       If the command add-target is invalid.
```
Comment (xzynos) [inline src/main/java/seedu/moneygowhere/parser/ConsoleParser.java:487]: Thanks for updating the Javadoc comments!

File: src/main/java/seedu/moneygowhere/parser/ConsoleParser.java [hunk 3]
```diff
@@ -68,6 +70,12 @@ public class ConsoleParser {
     public static final String CONSOLE_COMMAND_SORT_EXPENSE_TYPE_DATE = "date";
     public static final String CONSOLE_COMMAND_SORT_EXPENSE_ORDER_ASCENDING = "ascending";
     public static final String CONSOLE_COMMAND_SORT_EXPENSE_ORDER_DESCENDING = "descending";
+    public static final String CONSOLE_COMMAND_ADD_TARGET = "add-target";
+    public static final String CONSOLE_COMMAND_ADD_TARGET_ARGUMENT_NAME = "name";
+    public static final String CONSOLE_COMMAND_ADD_TARGET_ARGUMENT_DATE_TIME = "datetime";
+    public static final String CONSOLE_COMMAND_ADD_TARGET_ARGUMENT_DESCRIPTION = "description";
+    public static final String CONSOLE_COMMAND_ADD_TARGET_ARGUMENT_AMOUNT = "amount";
+    public static final String CONSOLE_COMMAND_ADD_TARGET_ARGUMENT_CURRENT_AMOUNT = "current amount";
```
Comment (xzynos) [inline src/main/java/seedu/moneygowhere/parser/ConsoleParser.java:78]: Good use of constant variables!

**Review by xzynos — APPROVED** [review]

File: src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java (hunk 1 above)
Comment (penguin-s) [reply src/main/java/seedu/moneygowhere/userinterface/ConsoleInterface.java:316]: Oh shoot, alright I'll add that in! Thank you Fang Jie!

**Pull request merged**